packaging/manage_react_native_packager.sh

```diff
@@ -0,0 +1,25 @@
+set -e -u -o pipefail # Fail on error
+
+: ${RN_DIR:?"Need to set RN_DIR"}
+
+cd $RN_DIR
+
+npm start &
+npm_cmd_pid=$!
+
+while [[ -z "$(pgrep -P $npm_cmd_pid)" ]]
```

Comment:
Can you add a process name to the `pgrep` as well? It seems like this could match a false positive if npm spawns a different child for some reason.